checker/checker_deprecation_test.go

```diff
@@ -22,8 +23,17 @@ func getDeprecationFile(file string) string {
 	return fmt.Sprintf("../data/deprecation/%s", file)
 }
 
+func singleCheckConfig(c checker.BackwardCompatibilityCheck) checker.BackwardCompatibilityCheckConfig {
+	return checker.BackwardCompatibilityCheckConfig{
+		Checks:              []checker.BackwardCompatibilityCheck{c},
+		MinSunsetBetaDays:   31,
+		MinSunsetStableDays: 180,
+		Localizer:           *localizations.New("en", "en"),
+	}
+}
```

Comment:
utility function to test checkers in isolation (e.g. in this checker_deprecation_test.go I noticed we're testing both APIDeprecationCheck  and APIRemovedCheck)